src/helpers.c

```diff
@@ -1250,3 +1251,63 @@ void warn_nosigcheck(const char *file)
 	warn("\n");
 	warn("THE SIGNATURE OF %s WILL NOT BE VERIFIED\n\n", file);
 }
+
+int write_into_file(char *file, char *content, char *mode)
```

Comment:
We have several file operations under sys. I think you can put this there

Reply:
ok